### PR TITLE
refine and add a more detail doc

### PR DESCRIPTION
**Pull request by july2993**
- drop depend on MySQL client: https://github.com/XuHuaiyu/tpccgen/commit/7f681102b42e33cedbbec04f1f493ebb692cc60b
- Use cobra for usability also fix will not quit if only generate CSV 
- Fix download URL of go-tpc: https://github.com/XuHuaiyu/tpccgen/commit/c03383231ced9e91de6578254055d773d2dd8c69
    - the using one just 404 not found, replace it as the newest available one in https://github.com/pingcap/go-tpc/releases
- Fix lightning instance number  is 2: https://github.com/XuHuaiyu/tpccgen/commit/dd1df277cb4213ed5e130bc607bc1670e99f1a29
    - the slowest table is order_line, so should generate it in one machine but not the table stock
- Add README_CN.md, a more detail doc

### CODE REVIEW

File: README_CN.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
+
+## 一键 TPC-C 造数导入工具说明
```
Comment (IANTHEREAL) [inline README_CN.md:2]: 补充一下性能测试数据吧，放在最前面

Comment (july2993) [reply README_CN.md:2]: https://github.com/XuHuaiyu/tpccgen/pull/8/commits/14b07df91f34004d97a8230567605c15aa32d47a

还是放最后吧， 这个其实跟这工具没关系。

**Comment by zhouqiang-cl** [comment]
@july2993 can you resolve the conflict?

**Comment by july2993** [comment]
> @july2993 can you resolve the conflict?

done

**Review by zhouqiang-cl — COMMENTED** [review]
@yeya24 PTAL

**Comment by yeya24** [comment]
Seems we need to update rename all `tpccgen` to `odin`.

**Comment by july2993** [comment]
> Seems we need to update rename all `tpccgen` to `odin`.

can you help do this rename after merging this if we do want to change the name as `odin`?

**Comment by yeya24** [comment]
> > Seems we need to update rename all `tpccgen` to `odin`.
> 
> can you help do this rename after merging this if we do want to change the name as `odin`?

Yes, I can help. But tbh `tpccgen` is a better name than `odin`, no :P?

**Pull request merged**